common/djangoapps/entitlements/filters.py

```diff
@@ -0,0 +1,41 @@
+from django_filters import rest_framework as filters
```

Comment:
The filters and serializers don't necessarily need to be versioned, but they should be in the `api` package.